devopsloft.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+publish: "disabled"
```

Comment:
Possibly add some comment on the Feature flag, not a must

Reply:
I added one in the README.md file